controller/engine_controller.go

```diff
@@ -107,6 +108,8 @@ func NewEngineController(
 	eventBroadcaster.StartRecordingToSink(&v1core.EventSinkImpl{Interface: v1core.New(kubeClient.CoreV1().RESTClient()).Events("")})
 
 	ec := &EngineController{
+		baseController: newBaseController("longhorn-engine", logger),
```

Comment:
just call it `engine` instead of `longhorn-engine`. It's an unnecessary prefix since it's produced by the Longhorn manager.

Same below.

Reply:
Agree, I kept the prior names but did think that the prefix is unnecessary.

Reply:
After discussion with @joshimoo , I realize the controller name is also used in the event recorder, which might cause confusion if we remove the `longhorn-` prefix it that. So I am good with keeping the `longhorn-` prefix and keep controller name same across different logging fields.